src/components/Quest.vue

```diff
@@ -2,7 +2,9 @@
 <div id="quest">
   <pie-chart id="counter" :numerator="countdownSeconds" :denominator="questionTimeoutSeconds" :textPercent="false" />
   <div class="container" v-for="(q, index) in questions" :key="q.newsId">
-    <transition name="question-transition" v-on:after-enter="afterEnter">
+    <transition name="question-transition"
```

Comment:
Skąd wzięły się tutaj te rzeczy? już powinny być dodane do mastera w jednym z poprzednich PR...

Reply:
@ppeczek Wygląda, że dodałeś po drodze merge commita - może on tu nabałaganił?
Czy jest możliwe żebyś po prostu zrobił pulla do `master`a i rebase do `master`a (zamiast merge, który produkuje niepotrzebnego commita) względem swojego brancha? np:
```shell
git pull --rebase origin master
git switch pp-answer-animation  # dla pewnośći
git rebase master
```

Reply:
o kurde. github mi powiedział że branch jest nieaktualny, żeby kliknąć tutej, to zrobi porządek. klikłem... :/